docs/dev/adding-documentation.md

```diff
@@ -0,0 +1,150 @@
+lines before a block is considered documentation:
+
+```cpp
+/// This line will be ignored
+int somefunc() { return 5; }
+
+///
+/// This line will be included
+///
+void otherfunc() { ; }
+```
+````
+If you wish to make a more visible comment block, e.g. a header for a
+class, then you may do something like the following:
+
+```cpp
+/********************************************//**
+* Here is some text inside a visible block
+***********************************************/
+```
+
+```{note}
+Doxygen will view this as any other documentation comment and
+will not render it any differently than a 'normal' documentation
+comment--it is simply more visible within the source code.
+```
+
+## How to include Doxygen's autodocs within Sphinx files
+
+Through the use of the Breathe extension it is incredibly easy to
+include Doxygen autodocs within a Sphinx documentation file.
+
+Suppose we have a C++ implementation file name `lily.h` that has
+inline comment documentation as detailed above and that `lily.h` is a
+component of a module named `flowers` that was just created.
+
+To document them, you must create a file within the Symbulation Library
+documentation source to hold the module's documentation:
+
+```cpp
+    touch doc/library/flowers.md
+```
+
+Within `flowers.md` you can make an introduction to the module,
+etc., and then add in the sphinx directives to include
+auto-documentation. Your `flowers.md` file should look something
+like the following:
+
+```md
+    # This is the flowers documentation!
+
+    This is a very short introduction.
+
+    ## lily.h
+
+    ```{eval-rst}
+    .. doxygenfile:: lily.h
+       :project: Symbulation
+    ```
+```
+
+When the docs are built Sphinx will automatically pull the available
+documentation from Doxygen's XML files to construct the docs.
+
+Additional directives exist to include auto-documentaiton from different
+levels, the full breakdown of which is available within the [Breathe
+Documentation](https://breathe.readthedocs.org/en/latest/directives.html).
+
+## How to add docs to the Sphinx documentation
+
+Sphinx is the system used to generate the developer guide and similar
+reference documentation. A primer to using ReStructured Text, the markup
+language used by Sphinx, can be found
+[here](http://docutils.sourceforge.net/docs/user/rst/quickstart.html).
+You can also look at any of the [.rst]{.title-ref} files in the
+[doc/]{.title-ref} directory to get a feel for how thinks work.
+
+New documents must be included in the `toctree` in the `index.md`
+file for the directory the added file lives in. For example, if you add
+`CowFacts.md` to the `CoolFacts/` directory you must add `CowFacts.md`
+to the toctree found in `CoolFacts/CowFacts.md`:
+
+```md
+    # Cool Facts
+
+    A bunch of cool facts!
+
+    ```{toctree}
+    AnteaterFacts
+    BirdFacts
+    CowFacts
+    ```
+```
+
+To build the documentation, we recommend you use our [Docker container](https://hub.docker.com/r/devosoft/Symbulation)
```

Comment:
This isn't correct, how have you been building the documentation? Just locally? For now you'll need to just cut out mention of the Docker container